components/schematics/ng-add/schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "CodebaseGenerator",
```

Comment:
Could you use a better id and description here, because i don't think the schematic creates a benchmark codebase.